packages/loader/src/loader.ts

```diff
@@ -26,7 +26,9 @@ export type CreateOptions = CreateOpts & {
   shouldIndex?: boolean
 }
 
-export type DeterministicLoadOptions = CreateOptions & {
+export type DeterministicLoadOptions = CreateOpts & {
+  controller?: string
```

Comment:
This new `controllers` property doesnt get used, is this for a future development?

Reply:
No it's not needed actually, good catch!